llvm/lib/Support/SpecialCaseList.cpp

```diff
@@ -71,9 +76,9 @@ unsigned SpecialCaseList::Matcher::match(StringRef Query) const {
   for (const auto &[Pattern, Pair] : Globs)
     if (Pair.first.match(Query))
       return Pair.second;
-  for (const auto &[Regex, LineNumber] : RegExes)
-    if (Regex->match(Query))
-      return LineNumber;
+  for (auto it = RegExes.crbegin(); it != RegExes.crend(); ++it)
```

Comment:
We need to change regex, but I expected my corner cases executed by Glob code.
Glob is default. How regex managed to fix 5 and 6?

Reply:
checked - `assert(RegExes.empty());` here never trigger on those tests

Reply:
> checked - assert(RegExes.empty()); here never trigger on those tests

Thanks for the finding. It should be matched by Glob now.